// @sapper/internal/manifest-client.mjs
// This file is generated by Sapper — do not edit it!
export { default as Root } from '../../../routes/_layout.svelte';
export { preload as root_preload } from './shared';
export { default as ErrorComponent } from '../../../routes/_error.svelte';

export const ignore = [/^\/auth\/register\/?$/, /^\/auth\/logout\/?$/, /^\/auth\/login\/?$/, /^\/auth\/save\/?$/, /^\/auth\/user\/?$/];

export const components = [
	{
		js: () => import("../../../routes/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:index.svelte__"
	},
	{
		js: () => import("../../../routes/register/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:register/index.svelte__"
	},
	{
		js: () => import("../../../routes/settings/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:settings/index.svelte__"
	},
	{
		js: () => import("../../../routes/article/[slug].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:article/[slug].svelte__"
	},
	{
		js: () => import("../../../routes/profile/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:profile/index.svelte__"
	},
	{
		js: () => import("../../../routes/profile/[user]/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:profile/[user]/index.svelte__"
	},
	{
		js: () => import("../../../routes/profile/[user]/[view].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:profile/[user]/[view].svelte__"
	},
	{
		js: () => import("../../../routes/editor/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:editor/index.svelte__"
	},
	{
		js: () => import("../../../routes/editor/[slug].svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:editor/[slug].svelte__"
	},
	{
		js: () => import("../../../routes/login/index.svelte"),
		css: "__SAPPER_CSS_PLACEHOLDER:login/index.svelte__"
	}
];

export const routes = (d => [
	{
		// index.svelte
		pattern: /^\/$/,
		parts: [
			{ i: 0 }
		]
	},

	{
		// register/index.svelte
		pattern: /^\/register\/?$/,
		parts: [
			{ i: 1 }
		]
	},

	{
		// settings/index.svelte
		pattern: /^\/settings\/?$/,
		parts: [
			{ i: 2 }
		]
	},

	{
		// article/[slug].svelte
		pattern: /^\/article\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 3, params: match => ({ slug: d(match[1]) }) }
		]
	},

	{
		// profile/index.svelte
		pattern: /^\/profile\/?$/,
		parts: [
			{ i: 4 }
		]
	},

	{
		// profile/[user]/index.svelte
		pattern: /^\/profile\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 5, params: match => ({ user: d(match[1]) }) }
		]
	},

	{
		// profile/[user]/[view].svelte
		pattern: /^\/profile\/([^\/]+?)\/([^\/]+?)\/?$/,
		parts: [
			null,
			null,
			{ i: 6, params: match => ({ user: d(match[1]), view: d(match[2]) }) }
		]
	},

	{
		// editor/index.svelte
		pattern: /^\/editor\/?$/,
		parts: [
			{ i: 7 }
		]
	},

	{
		// editor/[slug].svelte
		pattern: /^\/editor\/([^\/]+?)\/?$/,
		parts: [
			null,
			{ i: 8, params: match => ({ slug: d(match[1]) }) }
		]
	},

	{
		// login/index.svelte
		pattern: /^\/login\/?$/,
		parts: [
			{ i: 9 }
		]
	}
])(decodeURIComponent);

if (typeof window !== 'undefined') {
	import("/home/ball/prj/realword/node_modules/sapper/sapper-dev-client.js").then(client => {
		client.connect(10000);
	});
}
