// @sapper/internal/manifest-server.mjs
// This file is generated by Sapper — do not edit it!
import * as route_0 from "../../../routes/auth/register.js";
import * as route_1 from "../../../routes/auth/logout.js";
import * as route_2 from "../../../routes/auth/login.js";
import * as route_3 from "../../../routes/auth/save.js";
import * as route_4 from "../../../routes/auth/user.js";
import component_0 from "../../../routes/index.svelte";
import component_1 from "../../../routes/register/index.svelte";
import component_2 from "../../../routes/settings/index.svelte";
import component_3, { preload as preload_3 } from "../../../routes/article/[slug].svelte";
import component_4, { preload as preload_4 } from "../../../routes/profile/index.svelte";
import component_5, { preload as preload_5 } from "../../../routes/profile/[user]/index.svelte";
import component_6, { preload as preload_6 } from "../../../routes/profile/[user]/[view].svelte";
import component_7 from "../../../routes/editor/index.svelte";
import component_8, { preload as preload_8 } from "../../../routes/editor/[slug].svelte";
import component_9 from "../../../routes/login/index.svelte";
import root from "../../../routes/_layout.svelte";
import error from "../../../routes/_error.svelte";

const d = decodeURIComponent;

export const manifest = {
	server_routes: [
		{
			// auth/register.js
			pattern: /^\/auth\/register\/?$/,
			handlers: route_0,
			params: () => ({})
		},

		{
			// auth/logout.js
			pattern: /^\/auth\/logout\/?$/,
			handlers: route_1,
			params: () => ({})
		},

		{
			// auth/login.js
			pattern: /^\/auth\/login\/?$/,
			handlers: route_2,
			params: () => ({})
		},

		{
			// auth/save.js
			pattern: /^\/auth\/save\/?$/,
			handlers: route_3,
			params: () => ({})
		},

		{
			// auth/user.js
			pattern: /^\/auth\/user\/?$/,
			handlers: route_4,
			params: () => ({})
		}
	],

	pages: [
		{
			// index.svelte
			pattern: /^\/$/,
			parts: [
				{ name: "index", file: "index.svelte", component: component_0 }
			]
		},

		{
			// register/index.svelte
			pattern: /^\/register\/?$/,
			parts: [
				{ name: "register", file: "register/index.svelte", component: component_1 }
			]
		},

		{
			// settings/index.svelte
			pattern: /^\/settings\/?$/,
			parts: [
				{ name: "settings", file: "settings/index.svelte", component: component_2 }
			]
		},

		{
			// article/[slug].svelte
			pattern: /^\/article\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "article_$slug", file: "article/[slug].svelte", component: component_3, preload: preload_3, params: match => ({ slug: d(match[1]) }) }
			]
		},

		{
			// profile/index.svelte
			pattern: /^\/profile\/?$/,
			parts: [
				{ name: "profile", file: "profile/index.svelte", component: component_4, preload: preload_4 }
			]
		},

		{
			// profile/[user]/index.svelte
			pattern: /^\/profile\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "profile_$user", file: "profile/[user]/index.svelte", component: component_5, preload: preload_5, params: match => ({ user: d(match[1]) }) }
			]
		},

		{
			// profile/[user]/[view].svelte
			pattern: /^\/profile\/([^\/]+?)\/([^\/]+?)\/?$/,
			parts: [
				null,
				null,
				{ name: "profile_$user$93_$91view", file: "profile/[user]/[view].svelte", component: component_6, preload: preload_6, params: match => ({ user: d(match[1]), view: d(match[2]) }) }
			]
		},

		{
			// editor/index.svelte
			pattern: /^\/editor\/?$/,
			parts: [
				{ name: "editor", file: "editor/index.svelte", component: component_7 }
			]
		},

		{
			// editor/[slug].svelte
			pattern: /^\/editor\/([^\/]+?)\/?$/,
			parts: [
				null,
				{ name: "editor_$slug", file: "editor/[slug].svelte", component: component_8, preload: preload_8, params: match => ({ slug: d(match[1]) }) }
			]
		},

		{
			// login/index.svelte
			pattern: /^\/login\/?$/,
			parts: [
				{ name: "login", file: "login/index.svelte", component: component_9 }
			]
		}
	],

	root,
	root_preload: () => {},
	error
};

export const build_dir = "__sapper__/dev";

export const src_dir = "src";

export const dev = true;
